Sonification/MicroRhythm/app.js

```diff
@@ -839,7 +906,7 @@ const app = new Vue({
         }
     },
     mounted() {
-        this.setupDrag();
+        // this.setupDrag();
```

Comment:
Was this intended to be a temporary removal of the drag/drop feature?

Reply:
I thought my half-baked drag/drop feature (for CODAP attributes into plugin parameters) was not currently functional, so I disabled it. With the above uncommented, the user would see the parameter boxes visually highlighted when dragging an attribute in CODAP, but dropping into a highlighted parameter would not do anything. I thought this was rather misleading.

Reply:
Drag/drop is still functional for pitch and time attributes in version 0.2.8. I don't use it often. I don't know if others do. I wouldn't object to removing the feature and see if others notice...